pilot/pkg/proxy/envoy/discovery.go

```diff
@@ -184,6 +184,14 @@ type DiscoveryService struct {
 	sdsCache *discoveryCache
 
 	RestContainer *restful.Container
+
+	// edsUpdatedServices contains the list of Services that received EDS updates since
```

Comment:
do you mean `edsUpdatedServices` is the list of services updated since last eds push?